DEVELOPMENT.md

```diff
@@ -57,18 +57,23 @@ make test
 
 ### Integration tests
 
-On a GCP project where Kritis has already been installed, this will prepare a new cluster named `kritis-integration-test`:
+On a GCP project where Kritis has already been installed (follow [instructions](docs/install.md)), this will prepare a new cluster named `kritis-integration-test`:
 
 ```shell
-make -e GCP_PROJECT=<project id> setup-integration-local
+make -e GCP_PROJECT=${PROJECT_ID} setup-integration-local
 ```
 
+NOTE: the `PROJECT_ID` here is from the installation instructions environment
```

Comment:
This and elsewhere is not consistent with the name `$PROJECT` used in docs/install.md.

Reply:
Good point, fixed.